### DISCUSSION TITLE
自定义顶栏搜索栏，能否增加一个清空搜索栏的功能图标

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#4793] [功能建议 / Ideas]
自定义顶栏搜索栏，能否增加一个清空搜索栏的功能图标。
我觉得任何搜索栏，都应该有的功能，就是选定搜索栏，清除搜索栏内的文字，可以直接输入自己想要搜索的文字内容。
![小Q截图-20240615130134](https://github.com/the1812/Bilibili-Evolved/assets/64629623/ebcd3839-9fde-471d-8c71-e8611a560793)

**Comment by Geeyun-JY3** [comment]
我以前提过（：#801

**Reply by kethoven** [reply]
我觉得这是一个非常方便的小功能

**Reply by Geeyun-JY3** [reply]
我当然是支持的，我心中一个理想的搜索栏就该有这么个清空按钮（破站不也给自己搜索页面的搜索框加了），毕竟我个人平时用起来只用鼠标、只用键盘和键鼠混用都有可能，要清空搜索栏还得全选（鼠标三连击或按住鼠标左键滑动选择或 <kbd>Ctrl</kbd>+<kbd>A</kbd>）再删除多少还是麻烦了些。
